spdx/parsers/rdfbuilders.py

```diff
@@ -163,6 +164,8 @@ def create_entity(self, doc, value):
             return self.build_person(doc, value)
         elif self.org_re.match(value):
             return self.build_org(doc, value)
+        elif value == "NOASSERTION":            # Supplier name could be NOASSERTION SPDX 2.3
+            return  utils.NoAssert()   
```

Comment:
If you in include this check here, any field that is created through this method, so basically every entity (e.g. the creator of the document, any reviewer, any annotator) could also be set to `NoAssert` which is not valid in general.